### DISCUSSION TITLE
Mobile access cannot stream conversations

### DISCUSSION THREAD

**Discussion by StarRing2022** [open-webui/open-webui#6046] [General]
![微信图片_20241009131543](https://github.com/user-attachments/assets/e7ec2262-0d49-43bd-89cf-04384016ca22)
The current issue is that when accessing web pages on a mobile phone, the streaming conversation response feature needs to be turned off in order to have a normal conversation, while on a computer, streaming can be used normally. As streaming dialogue is one of the essential features for real-world applications, please take it seriously and fix it as soon as possible. Thank you!

**Comment by tjbck** [comment]
Cannot reproduce, check your network configuration.

**Comment by StarRing2022** [comment]
Thank you, you can access web services(pip install open-webui，http://127.0.0.1:8080) on the Safari browser on your iPhone. When using streaming conversations, it often only appears the first word and then reports an error. Invalid regular expression: Invalid specifier name，But it's strange that there won't be any errors on the computer
![微信图片_20241009221540](https://github.com/user-attachments/assets/af5f9a53-54af-42ac-bad0-f02ae57967b4)

**Comment by mb719** [comment]
Had a similar problem (one word replies) yesterday specifically just with iOS. I'll try turning off streaming and let you know!

**Reply by StarRing2022** [reply]
That's how I did it. I closed the streaming conversation, which was normal, but the problem is that in many practical products, if the non streaming waiting time is too long, users cannot accept it

**Reply by mb719** [reply]
> That's how I did it. I closed the streaming conversation, which was normal, but the problem is that in many practical products, if the non streaming waiting time is too long, users cannot accept it

LOL no kidding, I'm running with amd GPU which is like No GPU to Ollama, and I've definitely had to wait a minute before. I know that most people are getting addicted to instant gratification too!

**Reply by StarRing2022** [reply]
I have conducted the latest test and found that streaming on Huawei phones works normally. It seems that it is indeed an issue with Apple phones

**Reply by atgehrhardt** [reply]
Is it specific to any particular browser on iOS?

**Comment by StarRing2022** [comment]
Very interesting, it should be targeted at the entire device, including the
built-in browser in WeChat
